BUILD

```
@@ -63,6 +63,8 @@ container_bundle(
         "gcr.io/{PROJECT_ID}/java-debian10/jetty:java11-debug": "//java/jetty:jetty_java11_debug_debian10",
         "gcr.io/{PROJECT_ID}/python3:latest": "//experimental/python3:python3",
         "gcr.io/{PROJECT_ID}/python3:debug": "//experimental/python3:debug",
+        "gcr.io/{PROJECT_ID}/python3:nonroot": "//experimental/python3:python3-nonroot",
+        "gcr.io/{PROJECT_ID}/python3:debug-nonroot": "//experimental/python3:debug-nonroot",
```

It was our mistake that we used `-nonroot` instead of `_nonroot`. (I wasn't the one who reviewed the original PR that introduced `-nonroot`. For consistency, we should use `_` for Bazel rule names in this repo.

```suggestion
        "gcr.io/{PROJECT_ID}/python3:nonroot": "//experimental/python3:python3_nonroot",
        "gcr.io/{PROJECT_ID}/python3:debug_nonroot": "//experimental/python3:debug_nonroot",
```

---

Sure, sounds good to me